cmd/containerd-shim-runhcs-v1/task.go

```diff
@@ -110,6 +110,7 @@ func verifyTaskUpdateResourcesType(data interface{}) error {
 	case *specs.WindowsResources:
 	case *specs.LinuxResources:
 	case *ctrdtaskapi.PolicyFragment:
+	case *ctrdtaskapi.MappedPipe:
```

Comment:
I know we'll want to support other types in the future. I'm not sure if we will want to directly enumerate all the different additional resources we may want to support like this or if we'll want to create a type like `ExtendedWindowsContainerResources` or if we'll want to add the options directly to the runtime spec's structs themselves. I'm hesitant to move forward with this especially since we could potentially accidentally break you in the future. I'd like to hear more feedback from @kevpar or @ambarve here.

Reply:
I think this is fine. Even if we add more types in the future, I don't expect it would be difficult to continue supporting this type as well for update task calls.